### PR TITLE
Create da.lua

### PR DESCRIPTION
**Pull request by DuckoDas**
## Description
My pull request fixes or adds a new language, Danish, since a lot of new servers in Denmark are starting to use QBCore. I'm just adding a translation file for it. And that's why I want to help the Danish FiveM Community so they don't have to translate this.

## Checklist
- [x] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: locales/da.lua [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        is_staus = 'Er %{status}',
+        healthy = 'Du er helt frisk igen!',
+        safe = 'Hospital sikkerhed',
+        pb_hospital = 'Pillbox Hospital',
+        pain_message = 'Din %{limb} føles %{severity}',
+        many_places = 'Du har ondt mange steder...',
+        bleed_alert = 'Du er %{bleedstate}',
+        ems_alert = 'EMS alarm - %{text}',
+        mr = 'Hr.',
+        fru = 'Fru',
```
Comment (i-kulgu) [inline locales/da.lua:47]: must be : mrs = 'Fru',

File: locales/da.lua [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    },
+    mail = {
+        sender = 'Pillbox Hospital',
+        subject = 'Hospitalomkostninger',
+        message = 'Kære %{gender} %{lastname}, <br /><br />Hermed har du modtaget en mail med omkostningerne ved det sidste hospitalsbesøg.<br />De endelige omkostninger er blevet: <strong>%{costs} kr.</strong><br /><br />Vi ønsker dig en hurtig bedring!'
+    },
+    states = {
+        irritated = 'irriteret',
+        quite_painful = 'ganske smertefuldt',
+        smertefuld = 'smertefuld',
```
Comment (i-kulgu) [inline locales/da.lua:68]: must be : painful = 'smertefuld',

Comment (DuckoDas) [reply locales/da.lua:68]: Thanks

**Review by i-kulgu — APPROVED** [review]
Looks good to me.

**Review by zwkf — APPROVED** [review]
LGTM

**Pull request merged**